generated/java/gapic-google-cloud-error-reporting-v1beta1/src/main/java/com/google/cloud/errorreporting/spi/v1beta1/ErrorGroupServiceClient.java

```diff
@@ -87,7 +87,7 @@
  * </pre>
  */
 @Generated("by GAPIC")
-@ExperimentalApi
+@BetaApi
```

Comment:
Sure, the new clients requires updates on the gRPC release. The old structure depends on Maven so I will batch release a new version of gRPC/proto packages and create a separate PR after.